standalone-metastore/src/main/sql/mysql/hive-schema-3.0.0.mysql.sql

```diff
@@ -1078,12 +1078,13 @@ CREATE TABLE `I_SCHEMA` (
   `CAN_EVOLVE` bit(1) NOT NULL,
```

Comment:
It looks like you did updates to the mysql and postgres scripts but not the derby, oracle, and mssql scripts.  Those will need to be updated as well.